apps/api/src/api.module.ts

```diff
@@ -16,52 +16,84 @@ import { ConfigService } from '../../../libs/common/src/config/config.service';
 
 @Module({
   imports: [
-    BullModule.forRoot({
-      connection: {
-        enableOfflineQueue: false,
+    ConfigModule,
```

Comment:
apju module was missing proper declaration of BullMOdule since development controller Injects them, also organized the module nicely